coderd/prometheusmetrics/aggregator.go

```diff
@@ -8,8 +8,11 @@ import (
 	"time"
 
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/model"
 	"golang.org/x/xerrors"
 
+	"github.com/coder/coder/v2/coderd/agentmetrics"
```

Comment:
Unfortunately our formatter doesn't handle merging import groups and leaves things in a messy state (depending on what program injected them). 😔

If you notice these, please feel free to fix, but the standard is we try our best but sometimes these slip through, so don't worry too much.